src/Build/Resources/Strings.resx

```diff
@@ -1064,8 +1064,8 @@
     </comment>
   </data>
   <data name="TaskReturnedFalseButDidNotLogError">
-    <value>MSB4132: The "{0}" task returned false but did not log an error.</value>
-    <comment>{StrBegin="MSB4132: "}</comment>
+    <value>MSB4134: The "{0}" task returned false but did not log an error.</value>
```

Comment:
This appears to have been used, too:

```
MSB4134: DefaultToolsVersion cannot be set after a project has been loaded into the Engine.
```

That's coming entirely from the `Deprecated\Engine` folder, but we still shouldn't recycle.